spec/features/saml/loa1/account_creation_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+feature 'Canceling Account Creation' do
+  include SamlAuthHelper
+
+  context 'From the enter email page', email: true do
+    it 'redirects to the branded start page' do
+      authn_request = auth_request.create(saml_settings)
+      visit authn_request
+      sp_request_id = ServiceProviderRequest.last.uuid
+      click_link t('sign_up.registrations.create_account')
+      click_link t('links.cancel')
+
+      expect(current_url).to eq sign_up_start_url(request_id: sp_request_id)
```

Comment:
We could, but that's already tested elsewhere. The SP name is looked up based on the `request_id`. So, since we're already testing that the correct `request_id` is being included in the URL, we can safely assume the correct SP name will be displayed.